fs/subfs.cpp

```diff
@@ -228,48 +230,63 @@ namespace fs
             PathCat __(this, path);
             return underlayfs->mknod(path, mode, dev);
         }
-        /*
+
         virtual ssize_t getxattr(const char *path, const char *name, void *value, size_t size)
         {
+            if (!underlay_xattrfs)
+                LOG_ERROR_RETURN(ENOTSUP, -1, "xattr is not supported by underlay fs");
```

Comment:
Better to use `ENOSYS` as error number to present "method not implemented" as same as other not implemented situation in Photon.